startup/GafferImage/defaultColorSpace.py

```diff
@@ -0,0 +1,70 @@
+import PyOpenColorIO
+
+import GafferImage
+
+def defaultColorSpace( optionsData ):
+	"""
+	Default image colorSpace configuration,
+	based on file format options.
+
+	@param optionsData: IECore.CompounData CompoundData containing Image specs.
+	@retvalue string colorSpace name, valid for current OpenColorIO config.
+	"""
+	config = PyOpenColorIO.GetCurrentConfig()
+	sceneLinear = config.getColorSpace(PyOpenColorIO.Constants.ROLE_SCENE_LINEAR).getName()
+	log = config.getColorSpace(PyOpenColorIO.Constants.ROLE_COMPOSITING_LOG).getName()
+	display = config.getColorSpace(PyOpenColorIO.Constants.ROLE_COLOR_PICKING).getName()
+ 
+	colorSpaces = {
+		"openexr" : sceneLinear,
+
+		"exr" : sceneLinear,
+
+		"jpg" : display,
+
+		"dpx" : {
+			"uint8"  : display,
+			"uint16" : display,
+			"uint10" : log,
+			"uint12" : log,
+		},
+
+		"tiff" : {
+			"uint8"  : display,
+			"uint16" : display,
+			"float"  : sceneLinear,	#writing
+			"uint32"  : sceneLinear,	#reading
```

Comment:
Seems to be some misalignment of the comments here. Use spaces rather than tabs to align (and tabs to indent only).